mm2src/coins/eth/eth_hd_wallet.rs

```diff
@@ -31,7 +31,7 @@ impl ExtractExtendedPubkey for EthCoin {
 impl HDWalletCoinOps for EthCoin {
     type HDWallet = EthHDWallet;
 
-    fn address_formatter(&self) -> fn(&Address) -> String { display_eth_address }
+    fn address_formatter(&self) -> fn(&Address) -> String { DisplayAddress::display_address }
```

Comment:
Now that we have this `DisplayAddress` trait, I think we can remove `address_formatter`, let me give it a try.

Reply:
Done here https://github.com/KomodoPlatform/komodo-defi-framework/pull/2348/commits/055bca4fcfe6932e1000b691f0080f35b73d7ecb
@laruh @dimxy please give it a look